src/components/Form/Form.js

```diff
@@ -35,11 +35,11 @@ import { useFormState } from '../../hooks/useFormState'
  */
 
 export function Form({ children, config }) {
-  const inputNames = Object.keys(config.inputs)
+  const fieldNames = Object.keys(config.fields)
 
   // Set up initial values
   let initialValues = {}
-  inputNames.forEach((x) => {
+  fieldNames.forEach((x) => {
```

Comment:
Nitpick: `initialValues` could be set in a `reduce` statement instead a `forEach`, e.g.

```js
const initialValues = fieldNames.reduce((res, x) => {
  res[x] = `Initial invalid state for ${x}`
  return res; 
}, {})
```